src/patterns/SaleHeaderBanner/SaleHeaderBanner.tsx

```diff
@@ -0,0 +1,120 @@
+import { ComponentProps, forwardRef } from 'react';
+import { getCommonProps } from '../../utils';
+import classnames from 'classnames';
+import { SeldonImage } from '../../components/SeldonImage';
+import { AuctionState } from './types';
+import { Text, TextVariants } from '../../components/Text';
+import { PageContentWrapper as PageMargin } from '../../components/PageContentWrapper';
+import Button from '../../components/Button/Button';
+
+// You'll need to change the ComponentProps<"htmlelementname"> to match the top-level element of your component
+export interface SaleHeaderBannerProps extends ComponentProps<'div'> {
+  /**
+   * What is the title of the auction?
+   */
+  auctionTitle: string;
```

Comment:
can we make these `ReactNode` in case we need to pass custom elements (location, date, occuranceLabel and ctaLabel) probably as well